contracts/bonding/StakingManager.sol

```diff
@@ -28,39 +28,39 @@ contract StakingManager is ManagerProxyTarget, IStakingManager {
 
     struct Orchestrator {
         // Time-keeping
-        uint256 activationRound; // Round in which the transcoder became active - 0 if inactive
+        uint256 activationRound; // Round in which the orchestrator became active - 0 if inactive
         uint256 deactivationRound;
         // Commission accounting
         uint256 rewardShare; // % of reward shared with delegations
         uint256 feeShare; // % of fees shared with delegations
-        uint256 rewardCommissions; // reward earned from commission (not shared with delegators)
         uint256 feeCommissions; // fees earned from commission (not shared with delegators)
         uint256 lastRewardRound;
         // Delegation Pool
         Delegations.Pool delegationPool;
     }
 
-    // Represents an amount of tokens that are being unbonded
-    struct UnbondingLock {
+    // Represents an amount of tokens that are being undelegate
+    struct UnstakingLock {
+        address owner; // address which owns undelegated amount
         address orchestrator;
-        uint256 amount; // Amount of tokens being unbonded
-        uint256 withdrawRound; // Round at which unbonding period is over and tokens can be withdrawn
+        uint256 amount; // Amount of tokens being ustaked
+        uint256 withdrawRound; // Round at which undelegation period is over and tokens can be withdrawn
     }
 
-    // Time between unbonding and possible withdrawl in rounds
-    uint64 public unbondingPeriod;
+    // Time between unstaking and possible withdrawal in rounds
+    uint64 public unstakingPeriod;
```

Comment:
Let's convert this to a uint256 which saves a tiny bit of gas because under the hood it's converted to a uint256 anyway since everything is a uint256 in the EVM